### DISCUSSION TITLE
How to bookmark

### DISCUSSION THREAD

**Discussion by tkkcc** [kyoheiu/felix#225] [Q&A] [answered]
I can use zoxide to jump somewhere, but how to make zoxide remember a place (in felix)

**Comment by kyoheiu** [comment]
Sorry, I overlooked this discussion.

For jumping between directories I implemented `<C-o>` / `<C-i>` to jump backward / forward respectively, so app itself (kind of) `remembers` places.
And I realize that by using `zoxide-add` (https://man.archlinux.org/man/zoxide-add.1.en) felix itself can add or increment the rank of the directory. I'll give it a try.

Is this enough or you'd like to use a real bookmarking in this app, such as marking in Vim?

**Reply by tkkcc** [reply, marked as answer]
zoxide is enough. didn't know zoxide-add before.

**Comment by kyoheiu** [comment]
released in v2.8.0
